plugin-lifecycle/README.md

```diff
@@ -0,0 +1,41 @@
+## Mapbox Maps Lifecycle Plugin for Android
+
+### Overview
+
+The Mapbox Maps Lifecycle Plugin for Android is an public library for handling Lifecycle of MapView. By default, Lifecycle Plugin will automatically listen to the lifecycle events(onStart/onStop/onDestroy) of the host Activity or Fragment, so that users don't need to invoke the corresponding lifecycle methods of MapView in these three lifecycle methods.
+
+A full overview of classes and interfaces can be found in our [API documentation](https://docs.mapbox.com/android/beta/maps/guides/).
+
+### Getting Started
+
+This README is intended for developers who are interested in [contributing](https://github.com/mapbox/mapbox-maps-android/blob/master/CONTRIBUTING.md) to the Mapbox Maps Lifecycle Plugin for Android. Please visit [DEVELOPING.md](https://github.com/mapbox/mapbox-maps-android/blob/master/DEVELOPING.md) for general information and instructions on how to use the Mapbox Maps Plugin System. To add the Lifecycle plugin to your project, you configure its dependency in your `build.gradle` files.
+
+```groovy
+// In the root build.gradle file
+// The Mapbox access token needs to a scope set to DOWNLOADS:READ
+allprojects {
+    repositories {
+        maven {
+            url 'https://api.mapbox.com/downloads/v2/releases/maven'
+            authentication {
+                basic(BasicAuthentication)
+            }
+            credentials {
+                username = "mapbox"
+                password = "INSERT_MAPBOX_ACCESS_TOKEN_HERE"
+            }
+        }
+    }
+}
+
+// In the app build.gradle file
+dependencies {
+  implementation 'com.mapbox.plugin:maps-lifecycle:10.0.0-rc.4'
+  // Make sure the version of appcompat is 1.3.0+
```

Comment:
Did you think of putting `appcompat` dependency inside `lifecycle` plugin? Although I'm OK with current solution but just to make sure that idea was also given a shot.

Reply:
well, I've taken a look at gradle file of `lifecycle` plugin and appcompat is actually there. Do we need to specify it in dependency then?